armi/reactor/reactors.py

```diff
@@ -2269,6 +2269,19 @@ def processLoading(self, cs, dbLoad: bool = False):
                 )[0]
                 for a in self.parent.blueprints.assemblies.values():
                     a.makeAxialSnapList(refAssem=finestAssemblyMesh)
+            if not cs["inputHeightsConsideredHot"]:
+                self._axiallyExpandAssems(
+                    self.parent.blueprints.assemblies.values(),
+                    dbLoad,
+                    "blueprints",
+                    cs["detailedAxialExpansion"],
+                )
+                if not cs["detailedAxialExpansion"]:
+                    refMesh = (
+                        self.refAssem.getAxialMesh()
+                    )  # calculate once for efficiency
```

Comment:
with the way black formats, its sometimes nice to  move the comment to the line above. no action NEEDED though.

Reply:
Addressed https://github.com/terrapower/armi/pull/823/commits/de07353fcb87ce30d7af984de9d00e4199f1a408